core/src/main/java/jenkins/management/AdministrativeMonitorsConfiguration.java

```diff
@@ -43,7 +43,7 @@
     public boolean configure(StaplerRequest req, JSONObject json) throws FormException {
         for (AdministrativeMonitor am : AdministrativeMonitor.all()) {
             try {
-                boolean disable = !json.getJSONArray("administrativeMonitor").contains(am.id);
+                boolean disable = json.containsKey("administrativeMonitor") && !json.getJSONArray("administrativeMonitor").contains(am.id);
```

Comment:
It is not a complex fix. If a single Administrative monitor specified (JSONObject is not a JSONArray), there will be also an error (there us a bug for it IIRC)